### PR TITLE
Update alert Slack channel

### PR DESCRIPTION
**Pull request by mjsax**
Not sure if this is the right channel... Let me know if it should do somewhere else.

We want to eventually archive `#streams-alerts` channel.

### CODE REVIEW

**Review by guozhangwang — APPROVED** [review]

**Comment by apurvam** [comment]
If these are not high priority alerts please don’t send them to #ksql-alerts

Production pages go there and I’d like to keep noise out as much as possible

**Comment by ableegoldman** [comment]
Maybe we can create a low-priority alerts channel? Something like #ksql-notifications

**Comment by mjsax** [comment]
@vvcephei is driving a discussion about alerts (including DD and build/test failures etc). We can take it from there and can put this PR on hold in the mean time.

File: Jenkinsfile [hunk 1]
```diff
@@ -11,5 +11,5 @@ docker_oraclejdk8 {
     // nodeLabel = 'docker-oraclejdk8-compose'
     upstreamProjects = 'confluentinc/common'
     withPush = true
-    slackChannel = 'streams-alerts'
+    slackChannel = 'ksql-alerts'
```
Comment (mjsax) [inline Jenkinsfile:14]:
```suggestion
    slackChannel = 'ksqldb-warn'
```

**Comment by mjsax** [comment]
Update this PR to use `#ksqldb-warn` -- @confluentinc/ksql

**Review by vcrfxia — APPROVED** [review]

**Review by wcarlson5 — APPROVED** [review]

**Comment by cadonna** [comment]
Pint merged downstream

**Pull request merged**